src/ewatercycle/plugins/hype/forcing.py

```diff
@@ -124,3 +83,41 @@ def to_xarray(self) -> xr.Dataset:
             "history": "Created by ewatercycle.plugins.hype.forcing.HypeForcing.to_xarray()",
         }
         return ds
+
+
+def build_recipe(
+    start_year: int,
+    end_year: int,
+    shape: Path,
+    dataset: Dataset | str | dict,
+):
+    """Build an ESMValTool recipe for Hype forcing data.
+
+    Args:
+        start_year: The start year of the recipe.
+        end_year: The end year of the recipe.
+        shape: The shape of the recipe.
+        dataset: Dataset to get forcing data from.
+            When string is given a predefined dataset is looked up in
+            :py:const:`ewatercycle.esmvaltool.datasets.DATASETS`.
+            When dict given it is passed to
+            :py:class:`ewatercycle.esmvaltool.models.Dataset` constructor.
+
+    Returns:
+        The built recipe.
+    """
+    return (
+        RecipeBuilder()
+        .title("Hype forcing data")
+        .dataset(dataset)
+        .start(start_year)
+        .end(end_year)
+        .shape(shape, decomposed=True)
+        .lump()
+        .add_variable("tas", units="degC")
+        .add_variable("tasmin", units="degC")
+        .add_variable("tasmax", units="degC")
+        .add_variable("pr", units="kg m-2 d-1")
+        .script("hydrology/hype.py")
```

Comment:
I think it would be good to ship our own copies of the diagnostic scripts, or rather, have the plugins ship their own copies. Then we're not dependent on ESMValTool and everything that a model maintainer needs to maintain is in the same place.

Reply:
Done #381